obs-browser-plugin.cpp

```diff
@@ -142,7 +145,13 @@ static obs_properties_t *browser_source_get_properties(void *data)
 			obs_module_text("FPS"), 1, 60, 1);
 	obs_properties_add_text(props, "css",
 			obs_module_text("CSS"), OBS_TEXT_MULTILINE);
-	obs_properties_add_bool(props, "shutdown",
+    obs_properties_add_bool(props, "css_active",
```

Comment:
Style:  Spaces are used throughout this block for indentation instead of tabs.  Please fix these lines to use tabs for indentation like the surrounding lines.